unified-codes/apps/data-service/src/app/integration/v2_graphql.test.ts

```diff
@@ -0,0 +1,261 @@
+                ],
+              },
+              {
+                code: '945b5500',
+                description: 'Abacavir Oral Tablet',
+                type: 'form',
+                properties: null,
+                children: [
+                  {
+                    code: '3590a4bf',
+                    description: 'Abacavir Oral Tablet 300mg',
+                    type: 'strength',
+                    properties: null,
+                    children: null,
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      },
+    },
+  };
+
+  return fetch(url, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      query: `{
+  entity (code: "c7750265") {
+    code
+    description
+    type
+    properties {
+      type
+      value
+    }
+    children {
+      code
+      description
+      type
+      properties {
+        type
+        value
+      }
+      children {
+        code
+        description
+        type
+        properties {
+          type
+          value
+        }
+        children {
+          code
+          description
+          type
+          properties {
+            type
+            value
+          }
+          children {
+            code
+            description
+            type
+            properties {
+              type
+              value
+            }
+            children {
+              code
+              description
+              type
+              properties {
+                type
+                value
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}`,
+    }),
+  })
+    .then((res) => res.json())
+    .then((res) => {
+      expect(res.data.entity.code).toEqual('c7750265');
+      expect(res.data.entity.description).toEqual('Abacavir');
+      expect(JSON.stringify(res)).toContain('888e7a00');
+      expect(JSON.stringify(res)).toContain('db3d2000');
+      expect(JSON.stringify(res)).toContain('358b04bf');
+      expect(JSON.stringify(res)).toContain('945b5500');
+      expect(JSON.stringify(res)).toContain('3590a4bf');
+      expect(res.data.entity.properties).toEqual(expected.data.entity.properties);
+    });
+});
+
+test('Web UI Search - ace', () => {
+  const expected = {
+    data: {
+      entities: {
+        data: [
+          {
+            code: '33d71824',
+            description: 'Acetazolamide',
+            type: 'drug',
+            uid: '0x186d1',
+          },
+          {
+            code: '7c8c2b5b',
+            description: 'Acetic Acid',
+            type: 'drug',
+            uid: '0x186d5',
+          },
+          {
+            code: 'c8ba31a5',
+            description: 'Acetylcysteine',
+            type: 'drug',
+            uid: '0x186da',
+          },
+          {
+            code: 'cf00cc3f',
+            description: 'Acetylsalicylic Acid',
+            type: 'drug',
+            uid: '0x186e6',
```

Comment:
These uids were different for me, when I compared `res` and `expected`... can use `expect.any(String)` if you want to test the field exists though 👍

Reply:
I don't think we need to worry about uid, interesting that it's different every time. Makes it kind of useless to return...